app/src/main/java/ru/practicum/android/diploma/details/domain/impl/DetailRepository.kt

```diff
@@ -0,0 +1,12 @@
+package ru.practicum.android.diploma.details.domain.impl
```

Comment:
🍏 Кажется, интерфейс относится к пакету api)
```suggestion
package ru.practicum.android.diploma.details.domain.api
```